psyneulink/core/compositions/composition.py

```diff
@@ -10089,7 +10089,7 @@ def run(
                     elif execution_mode & pnlvm.ExecutionMode.PTX:
                         results += _comp_ex.cuda_run(inputs, num_trials, num_inputs_sets)
                     else:
-                        assert False, "Unknown execution mode: {}".format(execution_mode)
+                        assert False, f"Unknown execution mode: {execution_mode}."
```

Comment:
`.format` is preferred to f-strings at least in compilation related code.

Reply:
reverted